molecule/default/roles/ansible-mysqld_exporter

```diff
@@ -0,0 +1 @@
+../../../
```

Comment:
Why this file is needed?

Reply:
I was getting that error while trying to run the tests otherwise:

![image](https://user-images.githubusercontent.com/2412909/70714196-a1f3f700-1cb5-11ea-81ad-7a61708d44cf.png)

In the end, I can't run them because I'm on mac OSX

Reply:
The reason it failed is because your directory name is `fork-ansible-mysqld_exporter` instead of `ansible-mysqld_exporter`.

Reply:
Thanks for the for about the role name!

The tests still don't run though.
Might have something to do with the fact that cgroups are a linux thing: 
```
    volumes:
      - /sys/fs/cgroup:/sys/fs/cgroup:ro
```

```
--> Scenario: 'default'
--> Action: 'prepare'
    
    PLAY [Prepare] *****************************************************************
    
    TASK [Gathering Facts] *********************************************************
    fatal: [fedora]: UNREACHABLE! => {"changed": false, "msg": "Authentication or permission failure. In some cases, you may have been able to authenticate and did not have permissions on the target directory. Consider changing the remote tmp path in ansible.cfg to a path rooted in \"/tmp\". Failed command was: ( umask 77 && mkdir -p \"` echo unable to find user alexc: no matching entries in passwd file/.ansible/tmp/ansible-tmp-1576281939.684053-126070393443122 `\" && echo ansible-tmp-1576281939.684053-126070393443122=\"` echo unable to find user alexc: no matching entries in passwd file/.ansible/tmp/ansible-tmp-1576281939.684053-126070393443122 `\" ), exited with result 126, stdout output: unable to find user alexc: no matching entries in passwd file\r\n", "unreachable": true}
    fatal: [centos7]: UNREACHABLE! => {"changed": false, "msg": "Authentication or permission failure. In some cases, you may have been able to authenticate and did not have permissions on the target directory. Consider changing the remote tmp path in ansible.cfg to a path rooted in \"/tmp\". Failed command was: ( umask 77 && mkdir -p \"` echo unable to find user alexc: no matching entries in passwd file/.ansible/tmp/ansible-tmp-1576281939.686524-172252172000607 `\" && echo ansible-tmp-1576281939.686524-172252172000607=\"` echo unable to find user alexc: no matching entries in passwd file/.ansible/tmp/ansible-tmp-1576281939.686524-172252172000607 `\" ), exited with result 126, stdout output: unable to find user alexc: no matching entries in passwd file\r\n", "unreachable": true}
```

It's no big deal though. If this PR and https://github.com/cloudalchemy/ansible-alertmanager/pull/103 are good it'd be nice if you could merge!

Reply:
Nvm figured it out. Had to run as root.